src/cozmo/version.py

```diff
@@ -17,5 +17,5 @@
 # Specify the version of cozmoclad that this package requires
 # Releases of the Cozmo package must specify an exact cozmoclad release
 # to ensure compatibility with a specific release of the ios/android app.
-#__cozmoclad_version__ = None
-__cozmoclad_version__ = "1.0.1"
+__cozmoclad_version__ = None
```

Comment:
That will break with a clad mismatch on the next app update, i'll add a task that we need to document how to use the repo.